tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -1547,6 +1606,137 @@ describe("clipboard", () => {
     expect(getStyle(model, "C2")).toEqual({});
   });
 
+  test("copy cells with CF => remove origin CF => paste => it should paste with original CF", () => {
```

Comment:
In my opinion, we should aim to make tests short and readable, rather than try to test absolutely everything. I don't think you need to test that every single command is successfully dispatched, nor do you need to test both than the cell content is pasted AND the CF is pasted AND that the Cf succefully is applied. That's a lot of bloating, making the test hard to understand at first glance IMO.

The test could be 15 lines instead of 60, only adding a CF => copy it => Remove the CF => paste => check the CF is pasted. We don't need to also test that we can paste cell values, nor that CF are correctly applied to the cells. It's tested somewhere else.

2 small general details:
- try to prettify thing on single lines when you can. `expect(getStyle(model, "A1")).toEqual({fillColor: "#00FF00" });` is short enough to be on a single line, no need to take 3 lines
- if you want to test that a command behaved correctly, avoid testing `toBeSuccessfullyDispatched` but rather the state of the model after the command. For example here `REMOVE_CONDITIONAL_FORMAT`would have been dispatched, but not actually removed the CF. But again, you don't need to test that every single command is successful, I'd say to only test it if it helps the readability of the test.

Reply:
(same comment for the next test)